receiver/receivertest/nop_receiver.go

```diff
@@ -75,11 +75,3 @@ type nopReceiver struct {
 	component.StartFunc
 	component.ShutdownFunc
 }
-
-// NewNopBuilder returns a receiver.Builder that constructs nop receivers.
-func NewNopBuilder() *receiver.Builder {
```

Comment:
This has to stay as deprecated before we can remove it. PLease see https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md#breaking-changes